.github/workflows/release.yml

```diff
@@ -122,82 +122,107 @@ jobs:
+  get-smoke-test-configuration:
     runs-on: ubuntu-latest
     needs: generate-note
-    name: Smoke Test - ${{ matrix.aws_region }} - ${{ matrix.name }}
+    name: Set ${{ github.event.inputs.layer_kind }} Smoke Test Configuration Values
+    outputs:
+      sample_app_name: ${{ steps.set-smoke-test-outputs.outputs.sample_app_name }}
+      language: ${{ steps.set-smoke-test-outputs.outputs.language }}
+      build_directory: ${{ steps.set-smoke-test-outputs.outputs.build_directory }}
+      build_command: ${{ steps.set-smoke-test-outputs.outputs.build_command }}
+      terraform_directory: ${{ steps.set-smoke-test-outputs.outputs.terraform_directory }}
+      expected_trace_template: ${{ steps.set-smoke-test-outputs.outputs.expected_trace_template }}
+      expected_metric_template: ${{ steps.set-smoke-test-outputs.outputs.expected_metric_template }}
+      amp_regions: ${{ steps.set-smoke-test-outputs.outputs.amp_regions }}
+    steps:
+      - name: Configure Java Agent Values
+        if: ${{ github.event.inputs.layer_kind == 'java-agent' }}
+        run: |-
+          echo "SAMPLE_APP_NAME=java-awssdk-agent" >> $GITHUB_ENV;
+          echo "LANGUAGE=java" >> $GITHUB_ENV;
+          echo "BUILD_DIRECTORY=java" >> $GITHUB_ENV;
+          echo "BUILD_COMMAND=./build.sh" >> $GITHUB_ENV;
+          echo "TERRAFORM_DIRECTORY=sample-apps/java-agent-aws-sdk-terraform" >> $GITHUB_ENV;
+          echo "EXPECTED_TRACE_TEMPLATE=adot/utils/expected-templates/java-awssdk-agent.json" >> $GITHUB_ENV;
+          echo "EXPECTED_METRIC_TEMPLATE=adot/utils/expected-templates/java-awssdk-agent-metric.json" >> $GITHUB_ENV;
+          echo "AMP_REGIONS=us-west-2,us-east-1,us-east-2,eu-central-1,eu-west-1" >> $GITHUB_ENV;
+      - name: Configure Java Wrapper Values
+        if: ${{ github.event.inputs.layer_kind == 'java-wrapper' }}
+        run: |-
+          echo "SAMPLE_APP_NAME=java-awssdk-wrapper" >> $GITHUB_ENV;
+          echo "LANGUAGE=java" >> $GITHUB_ENV;
+          echo "BUILD_DIRECTORY=java" >> $GITHUB_ENV;
+          echo "BUILD_COMMAND=./build.sh" >> $GITHUB_ENV;
+          echo "TERRAFORM_DIRECTORY=sample-apps/java-wrapper-aws-sdk-terraform" >> $GITHUB_ENV;
+          echo "EXPECTED_TRACE_TEMPLATE=adot/utils/expected-templates/java-awssdk-wrapper.json" >> $GITHUB_ENV;
+      # # FIXME: (enowell) Do we need to have 2 different soak tests for Java Wrapper Layer?
+      # - name: Configure Java Wrapper 2 Values?
+      #   if: ${{ github.event.inputs.layer_kind == 'java-wrapper' }}
+      #   run: |-
+      #     echo "SAMPLE_APP_NAME=java-okhttp-wrapper" >> $GITHUB_ENV;
+      #     echo "LANGUAGE=java" >> $GITHUB_ENV;
+      #     echo "BUILD_DIRECTORY=java" >> $GITHUB_ENV;
+      #     echo "BUILD_COMMAND=./build.sh" >> $GITHUB_ENV;
+      #     echo "TERRAFORM_DIRECTORY=sample-apps/java-wrapper-okhttp-terraform" >> $GITHUB_ENV;
+      #     echo "EXPECTED_TRACE_TEMPLATE=adot/utils/expected-templates/java-okhttp-wrapper.json" >> $GITHUB_ENV;
```

Comment:
```suggestion
```

Reply:
Delete this comment block and updated the other comment explaining the actual issue.